app/src/models/action.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * A state representing the app computing whether a planned action will require
```

Comment:
I think I know the intent behind this enum by reading this but i'm not sure. could we add a couple examples to this comment of what "a planned action" is? (merges, rebases, etc)

Reply:
Let me kick this down the road to the next PR - the rebase case is similar to merge but might help with clarifying these docs.